modules/client/shared/src/main/scala/com/azavea/stac4s/api/client/ETag.scala

```diff
@@ -0,0 +1,15 @@
+package com.azavea.stac4s.api.client
+
+import eu.timepit.refined.types.string.NonEmptyString
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.refined._
+import io.circe.{Decoder, Encoder}
+
+import scala.annotation.nowarn
+
+case class ETag[T](entity: T, tag: Option[NonEmptyString])
+
+object ETag {
+  @nowarn implicit def encoderETag[T: Encoder]: Encoder[ETag[T]] = deriveEncoder
```

Comment:
what's the `@nowarn` suppressing here?

Reply:
@jisantuc %) it thinks that `Encoder` implicit is unused

Reply:
It is the same error as here: https://github.com/azavea/stac4s/blob/cc166cffee6867dae9a4fb08e6664cbc43c9be9e/modules/core/shared/src/main/scala/com/azavea/stac4s/ProductFieldNames.scala#L16 🤷